tf/examples/EMI-RNN/00_emi_lstm_example_automode_false.ipynb

```diff
@@ -0,0 +1,496 @@
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Loading the data\n",
+    "x_train, y_train = np.load('./HAR/48_16/x_train.npy'), np.load('./HAR/48_16/y_train.npy')\n",
+    "x_test, y_test = np.load('./HAR/48_16/x_test.npy'), np.load('./HAR/48_16/y_test.npy')\n",
+    "x_val, y_val = np.load('./HAR/48_16/x_val.npy'), np.load('./HAR/48_16/y_val.npy')\n",
+    "\n",
+    "# BAG_TEST, BAG_TRAIN, BAG_VAL represent bag_level labels. These are used for the label update\n",
+    "# step of EMI/MI RNN\n",
+    "BAG_TEST = np.argmax(y_test[:, 0, :], axis=1)\n",
+    "BAG_TRAIN = np.argmax(y_train[:, 0, :], axis=1)\n",
+    "BAG_VAL = np.argmax(y_val[:, 0, :], axis=1)\n",
+    "NUM_SUBINSTANCE = x_train.shape[1]\n",
+    "print(\"x_train shape is:\", x_train.shape)\n",
+    "print(\"y_train shape is:\", y_train.shape)\n",
+    "print(\"x_test shape is:\", x_val.shape)\n",
+    "print(\"y_test shape is:\", y_val.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Computation Graph\n",
+    "\n",
+    "![Parst Computation graph illustration](img/3PartsGraph.png)\n",
+    "\n",
+    "The *EMI-RNN* computation graph is constructed out of the following three mutually disjoint parts:\n",
+    "\n",
+    "1. `EMI_DataPipeline`: An efficient data input pipeline that using the Tensorflow Dataset API. This module ingests data compatible with EMI-RNN and provides two iterators for a batch of input data, $x$ and label $y$. \n",
+    "2. `EMI_RNN`: The 'abstract' `EMI-RNN` class defines the methods and attributes required for the forward computation graph. An implementation based on LSTM - `EMI_LSTM` is used in this document, though the user is free to implement his own computation graphs compatible with `EMI-RNN`. This module expects two Dataset API iterators for $x$-batch and $y$-batch as inputs and constructs the forward computation graph based on them. Every implementation of this class defines an `output` operation - the output of the forward computation graph.\n",
+    "3. `EMI_Trainer`: An instance of `EMI_Trainer` class which defines the loss functions and the training routine. This expects an `output` operator from an `EMI-RNN` implementation and attaches loss functions and training routines to it. \n",
+    "\n",
+    "To build the computation graph, we create an instance of all the above and then connect them together.\n",
+    "\n",
+    "Note that, the `EMI_BasicLSTM` class is an implementation that uses an LSTM cell and pushes the LSTM output at each step to a secondary classifier for classification. This secondary classifier is not implemented as part of `EMI_BasicLSTM` and is left to the user to define by overriding the `createExtendedGraph` method, and the `restoreExtendedgraph` method.\n",
+    "\n",
+    "Note that, to use your custom loss, overload `createLossOp` function and return the newly created operator. Use `automode=False` while creating the EMI_Trainer call.\n",
```

Comment:
This is redundant and incorrect the moment this PR is accepted. Please make this change.